handlers/user.go

```diff
@@ -69,5 +69,33 @@ func (h Handler) DeleteUser(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	helpers.ResponseWithJSON(w, http.StatusOK, result)
+}
+
+func (h Handler) UpdateUserByID(w http.ResponseWriter, r *http.Request) {
+	var user models.User
```

Comment:
tầng handler nên xài UserInput ( 1 struct chỉ chứa input - response của User) ko nên xài models dùng chung với các tầng khác

Reply:
ohhhh, got it a